app/views/users/_user.html.erb

```diff
@@ -1,5 +1,19 @@
 <li>
-  <h3> <%= user.first_name %></h3>
+  <h1> <%= gravatar_for user %> </h1>
+  <h3> <%= link_to user.first_name, user_path(user) %></h3>
   <h5> <%= user.email %> </h5>
   <p> <%= user.created_at %> </p>
+  <div>
+    <div class="sidebar_box">
+      <% unless Friendship.exists?(current_user, user) %>
```

Comment:
* Let's not display the link to send a friend request for the current_user.
We can update this line with `<% unless Friendship.exists?(current_user, user) || current_user %>`